### PR TITLE
Add "Slack ToDos Planner" Agent

### PR DESCRIPTION
**Pull request by cmin764**
Adds this Agent from the hackathon, a Slack ToDos Planner.

### CODE REVIEW

File: agents/slack-todos/README.md
Comment (cmin764) [inline agents/slack-todos/README.md:1]: Tested with Slack and Google Calendar from the gallery, but not with Todoist as well.

File: agents/slack-todos/actions/README.md
Comment (cmin764) [inline agents/slack-todos/actions/README.md:1]: Maybe we should write better in this main README @sunilgovind .

File: agents/slack-todos/actions/actions.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from pathlib import Path
+from sema4ai.actions import action
+from todoist_api_python.api import TodoistAPI
+from sema4ai.actions import Secret
+from dotenv import load_dotenv
+
+import json
+
+load_dotenv(Path(__file__).absolute().parent / "devdata" / ".env")
+DEV_TODOIST_ACCESS_TOKEN = Secret.model_validate(os.getenv("DEV_TODOIST_ACCESS_TOKEN", ""))
```
Comment (cmin764) [inline agents/slack-todos/actions/actions.py:19]: Would be cool to also add and commit a _devdata/.env.template_ file which exposes that so we just copy-paste that into an _.env_ and fill-in the secret.

File: agents/slack-todos/actions/actions.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+@action
```
Comment (cmin764) [inline agents/slack-todos/actions/actions.py:22]: Maybe adding a `is_consequential` as `False` decorator as well.

File: agents/slack-todos/actions/devdata/input_get_todoist_active_tasks.json
Comment (cmin764) [inline agents/slack-todos/actions/devdata/input_get_todoist_active_tasks.json:1]: Would be good to customize and test these input params.

**Review by cmin764 — COMMENTED** [review]
Self-review

**Review by sunilgovind — APPROVED** [review]

**Pull request merged**